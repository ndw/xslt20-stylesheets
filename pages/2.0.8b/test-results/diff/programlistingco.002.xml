<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B"/>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="R.1" class="article">
         <header deltaxml:deltaV2="A=B" class="article-titlepage">
            <h2>Article wrapper</h2>
         </header>
         <div deltaxml:deltaV2="A=B" class="toc">
            <p>
               <b>Table of Contents</b>
            </p>
            <ul class="toc">
               <li>
                  <span>
                     <a href="#R.1.2">1. imagedata/@fileref + numbering + sep.</a>
                  </span>
               </li>
               <li>
                  <span>
                     <a href="#R.1.3">2. inlinegraphic/@fileref + numbering</a>
                  </span>
               </li>
               <li>
                  <span>
                     <a href="#R.1.4">3. textdata/@fileref w/o numbering or sep</a>
                  </span>
               </li>
            </ul>
         </div>
         <section deltaxml:deltaV2="A!=B" id="R.1.2" class="section">
            <div deltaxml:deltaV2="A!=B" class="section-titlepage">
               <h3 deltaxml:deltaV2="A">1. imagedata/@fileref + numbering + sep.</h3><h2 deltaxml:deltaV2="B">1. imagedata/@fileref + numbering + sep.</h2>
            </div>
            <div deltaxml:deltaV2="A" class="programlisting">
               <pre>
                  <span class="linenumber">  1</span>
                  <span class="linenumber-separator"> &gt;</span>@rem = '--*-Perl-*--                                       <a name="prologue1"/>
                  <img src="http://localhost:8119/base/img/1.png" alt="1" border="0"/>
                  <span class="linenumber"/>
                  <span class="linenumber-separator"> &gt;</span>@echo off
<span class="linenumber"/>
                  <span class="linenumber-separator"> &gt;</span>perl.exe %_batchname %$
<span class="linenumber"/>
                  <span class="linenumber-separator"> &gt;</span>goto endofperl                                             <a name="skipeof1"/>
                  <img src="http://localhost:8119/base/img/2.png" alt="2" border="0"/>
                  <span class="linenumber">  5</span>
                  <span class="linenumber-separator"> &gt;</span>@rem ';
<span class="linenumber"/>
                  <span class="linenumber-separator"> &gt;</span>
                  <span class="linenumber"/>
                  <span class="linenumber-separator"> &gt;</span># Compress mail...
<span class="linenumber"/>
                  <span class="linenumber-separator"> &gt;</span>
                  <span class="linenumber"/>
                  <span class="linenumber-separator"> &gt;</span>require 'n:/home/nwalsh/lib/cygnus.pl';                    <a name="xreq1"/>
                  <img src="http://localhost:8119/base/img/3.png" alt="3" border="0"/>
                  <span class="linenumber"> 10</span>
                  <span class="linenumber-separator"> &gt;</span>require 'timelocal.pl';                                    <a name="require1"/>
                  <img src="http://localhost:8119/base/img/3.png" alt="3" border="0"/>
                  <span class="linenumber"/>
                  <span class="linenumber-separator"> &gt;</span>use Cwd;   <a name="use1"/>
                  <img src="http://localhost:8119/base/img/4.png" alt="4" border="0"/>
                  <span class="linenumber"/>
                  <span class="linenumber-separator"> &gt;</span>
                  <span class="linenumber"/>
                  <span class="linenumber-separator"> &gt;</span>select (STDERR); $| = 1;
<span class="linenumber"/>
                  <span class="linenumber-separator"> &gt;</span>select (STDOUT); $| = 1;
<span class="linenumber"> 15</span>
                  <span class="linenumber-separator"> &gt;</span>
                  <span class="linenumber"/>
                  <span class="linenumber-separator"> &gt;</span>@DIRS = ("/home/nwalsh/Mail");
<span class="linenumber"/>
                  <span class="linenumber-separator"> &gt;</span>while (@DIRS) {
<span class="linenumber"/>
                  <span class="linenumber-separator"> &gt;</span>    $dir = shift @DIRS;
<span class="linenumber"/>
                  <span class="linenumber-separator"> &gt;</span>    opendir (DIR, $dir);
<span class="linenumber"> 20</span>
                  <span class="linenumber-separator"> &gt;</span>    while ($fname = readdir(DIR)) {
<span class="linenumber"/>
                  <span class="linenumber-separator"> &gt;</span>        $file = "$dir/$fname";
<span class="linenumber"/>
                  <span class="linenumber-separator"> &gt;</span>        next if ! -d $file;
<span class="linenumber"/>
                  <span class="linenumber-separator"> &gt;</span>        next if $fname =~ /^\.\.?$/;
<span class="linenumber"/>
                  <span class="linenumber-separator"> &gt;</span>
                  <span class="linenumber"> 25</span>
                  <span class="linenumber-separator"> &gt;</span>        print "$file\n";
<span class="linenumber"/>
                  <span class="linenumber-separator"> &gt;</span>        push (@DIRS, $file);
<span class="linenumber"/>
                  <span class="linenumber-separator"> &gt;</span>        &amp;compress ($file);                                 <a name="funccall1"/>
                  <img src="http://localhost:8119/base/img/5.png" alt="5" border="0"/>
                  <span class="linenumber"/>
                  <span class="linenumber-separator"> &gt;</span>    }
<span class="linenumber"/>
                  <span class="linenumber-separator"> &gt;</span>}
<span class="linenumber"> 30</span>
                  <span class="linenumber-separator"> &gt;</span>
                  <span class="linenumber"/>
                  <span class="linenumber-separator"> &gt;</span>exit;</pre>
            </div><pre deltaxml:deltaV2="B" data-line="1,4,9,10,11,27" class="programlisting language-none">
               <code>@rem = '--*-Perl-*--
@echo off
perl.exe %_batchname %$
goto endofperl
@rem ';

# Compress mail...

require 'n:/home/nwalsh/lib/cygnus.pl';
require 'timelocal.pl';
use Cwd;

select (STDERR); $| = 1;
select (STDOUT); $| = 1;

@DIRS = ("/home/nwalsh/Mail");
while (@DIRS) {
    $dir = shift @DIRS;
    opendir (DIR, $dir);
    while ($fname = readdir(DIR)) {
        $file = "$dir/$fname";
        next if ! -d $file;
        next if $fname =~ /^\.\.?$/;

        print "$file\n";
        push (@DIRS, $file);
        &amp;compress ($file);
    }
}

exit;</code>
            </pre>
         </section>
         <section deltaxml:deltaV2="A!=B" id="R.1.3" class="section">
            <div deltaxml:deltaV2="A!=B" class="section-titlepage">
               <h3 deltaxml:deltaV2="A">2. inlinegraphic/@fileref + numbering</h3><h2 deltaxml:deltaV2="B">2. inlinegraphic/@fileref + numbering</h2>
            </div>
            <div deltaxml:deltaV2="A" class="programlisting">
               <pre>
                  <span class="linenumber">  1</span>
                  <span class="linenumber-separator"/>@rem = '--*-Perl-*--                                       <a name="prologue2"/>
                  <img src="http://localhost:8119/base/img/1.png" alt="1" border="0"/>
                  <span class="linenumber">  2</span>
                  <span class="linenumber-separator"/>@echo off
<span class="linenumber"/>
                  <span class="linenumber-separator"/>perl.exe %_batchname %$
<span class="linenumber">  4</span>
                  <span class="linenumber-separator"/>goto endofperl                                             <a name="skipeof2"/>
                  <img src="http://localhost:8119/base/img/2.png" alt="2" border="0"/>
                  <span class="linenumber"/>
                  <span class="linenumber-separator"/>@rem ';
<span class="linenumber">  6</span>
                  <span class="linenumber-separator"/>
                  <span class="linenumber"/>
                  <span class="linenumber-separator"/># Compress mail...
<span class="linenumber">  8</span>
                  <span class="linenumber-separator"/>
                  <span class="linenumber"/>
                  <span class="linenumber-separator"/>require 'n:/home/nwalsh/lib/cygnus.pl';                    <a name="xreq2"/>
                  <img src="http://localhost:8119/base/img/3.png" alt="3" border="0"/>
                  <span class="linenumber"> 10</span>
                  <span class="linenumber-separator"/>require 'timelocal.pl';                                    <a name="require22"/>
                  <img src="http://localhost:8119/base/img/3.png" alt="3" border="0"/>
                  <span class="linenumber"/>
                  <span class="linenumber-separator"/>use Cwd;   <a name="use2"/>
                  <img src="http://localhost:8119/base/img/4.png" alt="4" border="0"/>
                  <span class="linenumber"> 12</span>
                  <span class="linenumber-separator"/>
                  <span class="linenumber"/>
                  <span class="linenumber-separator"/>select (STDERR); $| = 1;
<span class="linenumber"> 14</span>
                  <span class="linenumber-separator"/>select (STDOUT); $| = 1;
<span class="linenumber"/>
                  <span class="linenumber-separator"/>
                  <span class="linenumber"> 16</span>
                  <span class="linenumber-separator"/>@DIRS = ("/home/nwalsh/Mail");
<span class="linenumber"/>
                  <span class="linenumber-separator"/>while (@DIRS) {
<span class="linenumber"> 18</span>
                  <span class="linenumber-separator"/>    $dir = shift @DIRS;
<span class="linenumber"/>
                  <span class="linenumber-separator"/>    opendir (DIR, $dir);
<span class="linenumber"> 20</span>
                  <span class="linenumber-separator"/>    while ($fname = readdir(DIR)) {
<span class="linenumber"/>
                  <span class="linenumber-separator"/>        $file = "$dir/$fname";
<span class="linenumber"> 22</span>
                  <span class="linenumber-separator"/>        next if ! -d $file;
<span class="linenumber"/>
                  <span class="linenumber-separator"/>        next if $fname =~ /^\.\.?$/;
<span class="linenumber"> 24</span>
                  <span class="linenumber-separator"/>
                  <span class="linenumber"/>
                  <span class="linenumber-separator"/>        print "$file\n";
<span class="linenumber"> 26</span>
                  <span class="linenumber-separator"/>        push (@DIRS, $file);
<span class="linenumber"/>
                  <span class="linenumber-separator"/>        &amp;compress ($file);                                 <a name="funccall2"/>
                  <img src="http://localhost:8119/base/img/5.png" alt="5" border="0"/>
                  <span class="linenumber"> 28</span>
                  <span class="linenumber-separator"/>    }
<span class="linenumber"/>
                  <span class="linenumber-separator"/>}
<span class="linenumber"> 30</span>
                  <span class="linenumber-separator"/>
                  <span class="linenumber"/>
                  <span class="linenumber-separator"/>exit;</pre>
            </div><pre deltaxml:deltaV2="B" data-line="1,4,9,10,11,27" class="programlisting language-none">
               <code>@rem = '--*-Perl-*--
@echo off
perl.exe %_batchname %$
goto endofperl
@rem ';

# Compress mail...

require 'n:/home/nwalsh/lib/cygnus.pl';
require 'timelocal.pl';
use Cwd;

select (STDERR); $| = 1;
select (STDOUT); $| = 1;

@DIRS = ("/home/nwalsh/Mail");
while (@DIRS) {
    $dir = shift @DIRS;
    opendir (DIR, $dir);
    while ($fname = readdir(DIR)) {
        $file = "$dir/$fname";
        next if ! -d $file;
        next if $fname =~ /^\.\.?$/;

        print "$file\n";
        push (@DIRS, $file);
        &amp;compress ($file);
    }
}

exit;</code>
            </pre>
         </section>
         <section deltaxml:deltaV2="A!=B" id="R.1.4" class="section">
            <div deltaxml:deltaV2="A!=B" class="section-titlepage">
               <h3 deltaxml:deltaV2="A">3. textdata/@fileref w/o numbering or sep</h3><h2 deltaxml:deltaV2="B">3. textdata/@fileref w/o numbering or sep</h2>
            </div>
            <div deltaxml:deltaV2="A" class="programlisting">
               <pre>@rem = '--*-Perl-*--                                       <a name="prologue3"/>
                  <img src="http://localhost:8119/base/img/1.png" alt="1" border="0"/>
@echo off
perl.exe %_batchname %$
goto endofperl                                             <a name="skipeof3"/>
                  <img src="http://localhost:8119/base/img/2.png" alt="2" border="0"/>
@rem ';

# Compress mail...

require 'n:/home/nwalsh/lib/cygnus.pl';                    <a name="xreq3"/>
                  <img src="http://localhost:8119/base/img/3.png" alt="3" border="0"/>
require 'timelocal.pl';                                    <a name="require3"/>
                  <img src="http://localhost:8119/base/img/3.png" alt="3" border="0"/>
use Cwd;   <a name="use3"/>
                  <img src="http://localhost:8119/base/img/4.png" alt="4" border="0"/>

select (STDERR); $| = 1;
select (STDOUT); $| = 1;

@DIRS = ("/home/nwalsh/Mail");
while (@DIRS) {
    $dir = shift @DIRS;
    opendir (DIR, $dir);
    while ($fname = readdir(DIR)) {
        $file = "$dir/$fname";
        next if ! -d $file;
        next if $fname =~ /^\.\.?$/;

        print "$file\n";
        push (@DIRS, $file);
        &amp;compress ($file);                                 <a name="funccall3"/>
                  <img src="http://localhost:8119/base/img/5.png" alt="5" border="0"/>
    }
}

exit;</pre>
            </div><pre deltaxml:deltaV2="B" data-line="1,4,9,10,11,27" class="programlisting">
               <code>@rem = '--*-Perl-*--
@echo off
perl.exe %_batchname %$
goto endofperl
@rem ';

# Compress mail...

require 'n:/home/nwalsh/lib/cygnus.pl';
require 'timelocal.pl';
use Cwd;

select (STDERR); $| = 1;
select (STDOUT); $| = 1;

@DIRS = ("/home/nwalsh/Mail");
while (@DIRS) {
    $dir = shift @DIRS;
    opendir (DIR, $dir);
    while ($fname = readdir(DIR)) {
        $file = "$dir/$fname";
        next if ! -d $file;
        next if $fname =~ /^\.\.?$/;

        print "$file\n";
        push (@DIRS, $file);
        &amp;compress ($file);
    }
}

exit;</code>
            </pre>
         </section>
      </article>
   </body>
</html>