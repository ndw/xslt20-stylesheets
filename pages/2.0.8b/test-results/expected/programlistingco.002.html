<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Article wrapper</title><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.2" /><link rel="stylesheet" type="text/css" href="http://localhost:8119/base/css/default.css" /><script type="text/javascript" src="http://localhost:8119/base/js/dbmodnizr.js"></script></head><body><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#R.1.2">1. imagedata/@fileref + numbering + sep.</a></span></li><li><span><a href="#R.1.3">2. inlinegraphic/@fileref + numbering</a></span></li><li><span><a href="#R.1.4">3. textdata/@fileref w/o numbering or sep</a></span></li></ul></div><section id="R.1.2" class="section"><div class="section-titlepage"><h3>1. imagedata/@fileref + numbering + sep.</h3></div><div class="programlisting"><pre><span class="linenumber">  1</span><span class="linenumber-separator"> &gt;</span>@rem = '--*-Perl-*--                                       <a name="prologue1"></a><img src="http://localhost:8119/base/img/1.png" alt="1" border="0" />
<span class="linenumber">   </span><span class="linenumber-separator"> &gt;</span>@echo off
<span class="linenumber">   </span><span class="linenumber-separator"> &gt;</span>perl.exe %_batchname %$
<span class="linenumber">   </span><span class="linenumber-separator"> &gt;</span>goto endofperl                                             <a name="skipeof1"></a><img src="http://localhost:8119/base/img/2.png" alt="2" border="0" />
<span class="linenumber">  5</span><span class="linenumber-separator"> &gt;</span>@rem ';
<span class="linenumber">   </span><span class="linenumber-separator"> &gt;</span>
<span class="linenumber">   </span><span class="linenumber-separator"> &gt;</span># Compress mail...
<span class="linenumber">   </span><span class="linenumber-separator"> &gt;</span>
<span class="linenumber">   </span><span class="linenumber-separator"> &gt;</span>require 'n:/home/nwalsh/lib/cygnus.pl';                    <a name="xreq1"></a><img src="http://localhost:8119/base/img/3.png" alt="3" border="0" />
<span class="linenumber"> 10</span><span class="linenumber-separator"> &gt;</span>require 'timelocal.pl';                                    <a name="require1"></a><img src="http://localhost:8119/base/img/3.png" alt="3" border="0" />
<span class="linenumber">   </span><span class="linenumber-separator"> &gt;</span>use Cwd;   <a name="use1"></a><img src="http://localhost:8119/base/img/4.png" alt="4" border="0" />
<span class="linenumber">   </span><span class="linenumber-separator"> &gt;</span>
<span class="linenumber">   </span><span class="linenumber-separator"> &gt;</span>select (STDERR); $| = 1;
<span class="linenumber">   </span><span class="linenumber-separator"> &gt;</span>select (STDOUT); $| = 1;
<span class="linenumber"> 15</span><span class="linenumber-separator"> &gt;</span>
<span class="linenumber">   </span><span class="linenumber-separator"> &gt;</span>@DIRS = ("/home/nwalsh/Mail");
<span class="linenumber">   </span><span class="linenumber-separator"> &gt;</span>while (@DIRS) {
<span class="linenumber">   </span><span class="linenumber-separator"> &gt;</span>    $dir = shift @DIRS;
<span class="linenumber">   </span><span class="linenumber-separator"> &gt;</span>    opendir (DIR, $dir);
<span class="linenumber"> 20</span><span class="linenumber-separator"> &gt;</span>    while ($fname = readdir(DIR)) {
<span class="linenumber">   </span><span class="linenumber-separator"> &gt;</span>        $file = "$dir/$fname";
<span class="linenumber">   </span><span class="linenumber-separator"> &gt;</span>        next if ! -d $file;
<span class="linenumber">   </span><span class="linenumber-separator"> &gt;</span>        next if $fname =~ /^\.\.?$/;
<span class="linenumber">   </span><span class="linenumber-separator"> &gt;</span>
<span class="linenumber"> 25</span><span class="linenumber-separator"> &gt;</span>        print "$file\n";
<span class="linenumber">   </span><span class="linenumber-separator"> &gt;</span>        push (@DIRS, $file);
<span class="linenumber">   </span><span class="linenumber-separator"> &gt;</span>        &amp;compress ($file);                                 <a name="funccall1"></a><img src="http://localhost:8119/base/img/5.png" alt="5" border="0" />
<span class="linenumber">   </span><span class="linenumber-separator"> &gt;</span>    }
<span class="linenumber">   </span><span class="linenumber-separator"> &gt;</span>}
<span class="linenumber"> 30</span><span class="linenumber-separator"> &gt;</span>
<span class="linenumber">   </span><span class="linenumber-separator"> &gt;</span>exit;</pre></div></section><section id="R.1.3" class="section"><div class="section-titlepage"><h3>2. inlinegraphic/@fileref + numbering</h3></div><div class="programlisting"><pre><span class="linenumber">  1</span><span class="linenumber-separator"> </span>@rem = '--*-Perl-*--                                       <a name="prologue2"></a><img src="http://localhost:8119/base/img/1.png" alt="1" border="0" />
<span class="linenumber">  2</span><span class="linenumber-separator"> </span>@echo off
<span class="linenumber">   </span><span class="linenumber-separator"> </span>perl.exe %_batchname %$
<span class="linenumber">  4</span><span class="linenumber-separator"> </span>goto endofperl                                             <a name="skipeof2"></a><img src="http://localhost:8119/base/img/2.png" alt="2" border="0" />
<span class="linenumber">   </span><span class="linenumber-separator"> </span>@rem ';
<span class="linenumber">  6</span><span class="linenumber-separator"> </span>
<span class="linenumber">   </span><span class="linenumber-separator"> </span># Compress mail...
<span class="linenumber">  8</span><span class="linenumber-separator"> </span>
<span class="linenumber">   </span><span class="linenumber-separator"> </span>require 'n:/home/nwalsh/lib/cygnus.pl';                    <a name="xreq2"></a><img src="http://localhost:8119/base/img/3.png" alt="3" border="0" />
<span class="linenumber"> 10</span><span class="linenumber-separator"> </span>require 'timelocal.pl';                                    <a name="require22"></a><img src="http://localhost:8119/base/img/3.png" alt="3" border="0" />
<span class="linenumber">   </span><span class="linenumber-separator"> </span>use Cwd;   <a name="use2"></a><img src="http://localhost:8119/base/img/4.png" alt="4" border="0" />
<span class="linenumber"> 12</span><span class="linenumber-separator"> </span>
<span class="linenumber">   </span><span class="linenumber-separator"> </span>select (STDERR); $| = 1;
<span class="linenumber"> 14</span><span class="linenumber-separator"> </span>select (STDOUT); $| = 1;
<span class="linenumber">   </span><span class="linenumber-separator"> </span>
<span class="linenumber"> 16</span><span class="linenumber-separator"> </span>@DIRS = ("/home/nwalsh/Mail");
<span class="linenumber">   </span><span class="linenumber-separator"> </span>while (@DIRS) {
<span class="linenumber"> 18</span><span class="linenumber-separator"> </span>    $dir = shift @DIRS;
<span class="linenumber">   </span><span class="linenumber-separator"> </span>    opendir (DIR, $dir);
<span class="linenumber"> 20</span><span class="linenumber-separator"> </span>    while ($fname = readdir(DIR)) {
<span class="linenumber">   </span><span class="linenumber-separator"> </span>        $file = "$dir/$fname";
<span class="linenumber"> 22</span><span class="linenumber-separator"> </span>        next if ! -d $file;
<span class="linenumber">   </span><span class="linenumber-separator"> </span>        next if $fname =~ /^\.\.?$/;
<span class="linenumber"> 24</span><span class="linenumber-separator"> </span>
<span class="linenumber">   </span><span class="linenumber-separator"> </span>        print "$file\n";
<span class="linenumber"> 26</span><span class="linenumber-separator"> </span>        push (@DIRS, $file);
<span class="linenumber">   </span><span class="linenumber-separator"> </span>        &amp;compress ($file);                                 <a name="funccall2"></a><img src="http://localhost:8119/base/img/5.png" alt="5" border="0" />
<span class="linenumber"> 28</span><span class="linenumber-separator"> </span>    }
<span class="linenumber">   </span><span class="linenumber-separator"> </span>}
<span class="linenumber"> 30</span><span class="linenumber-separator"> </span>
<span class="linenumber">   </span><span class="linenumber-separator"> </span>exit;</pre></div></section><section id="R.1.4" class="section"><div class="section-titlepage"><h3>3. textdata/@fileref w/o numbering or sep</h3></div><div class="programlisting"><pre>@rem = '--*-Perl-*--                                       <a name="prologue3"></a><img src="http://localhost:8119/base/img/1.png" alt="1" border="0" />
@echo off
perl.exe %_batchname %$
goto endofperl                                             <a name="skipeof3"></a><img src="http://localhost:8119/base/img/2.png" alt="2" border="0" />
@rem ';

# Compress mail...

require 'n:/home/nwalsh/lib/cygnus.pl';                    <a name="xreq3"></a><img src="http://localhost:8119/base/img/3.png" alt="3" border="0" />
require 'timelocal.pl';                                    <a name="require3"></a><img src="http://localhost:8119/base/img/3.png" alt="3" border="0" />
use Cwd;   <a name="use3"></a><img src="http://localhost:8119/base/img/4.png" alt="4" border="0" />

select (STDERR); $| = 1;
select (STDOUT); $| = 1;

@DIRS = ("/home/nwalsh/Mail");
while (@DIRS) {
    $dir = shift @DIRS;
    opendir (DIR, $dir);
    while ($fname = readdir(DIR)) {
        $file = "$dir/$fname";
        next if ! -d $file;
        next if $fname =~ /^\.\.?$/;

        print "$file\n";
        push (@DIRS, $file);
        &amp;compress ($file);                                 <a name="funccall3"></a><img src="http://localhost:8119/base/img/5.png" alt="5" border="0" />
    }
}

exit;</pre></div></section></article></body></html>